src/components/createResidentPermit/PermitInfo.tsx

```diff
@@ -25,11 +25,12 @@ const PermitInfo = ({
   onUpdateField,
 }: PermitInfoProps): React.ReactElement => {
   const { t, i18n } = useTranslation();
-  const expirationDate = permit
-    ? formatDateTimeDisplay(
-        addMonths(new Date(permit.startTime), permit.monthCount)
-      )
-    : '';
+  const expirationDate = addDays(
+    addMonths(new Date(permit.startTime), permit.monthCount),
+    -1
+  );
+  // eslint-disable-next-line no-magic-numbers
+  expirationDate.setHours(23, 59, 0);
```

Comment:
This can be done:
`import { endOfDay } from 'date-fns';`
`endOfDay(expirationDate)';`

and then you can get rid of those magic numbers.

Reply:
Nice tip, thanks @amanpdyadav